entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnodeImpl.java

```diff
@@ -20,32 +20,38 @@
 
     private float xdp; // x divider position 1 -> 2
 
+    private String line1Name;
+
+    private boolean line1Fictitious;
+
     private double xnodeP1;
 
     private double xnodeQ1;
 
+    private float b1dp; // b1 divider position 1 -> 2
+
+    private float g1dp; // g1 divider position 1 -> 2
+
+    private String line2Name;
+
+    private boolean line2Fictitious;
+
     private double xnodeP2;
 
     private double xnodeQ2;
 
-    private String line1Name;
+    private float b2dp; // b2 divider position 1 -> 2
 
-    private String line2Name;
+    private float g2dp; // g2 divider position 1 -> 2
 
     private String code;
 
-    public MergedXnodeImpl(Line line, float rdp, float xdp, double xnodeP1, double xnodeQ1, double xnodeP2, double xnodeQ2,
-                           String line1Name, String line2Name, String code) {
+    public static MergedXnode create(Line line) {
```

Comment:
I would rather have a constructor, it clashes a bit with our other extensions' implementations. Furthermore, it creates the risk to have an object with unset attributes which does not make sense for `rdp`, `xdp` or `code` for example

Reply:
I have created a builder to test the missing values.
I have deleted the constructor to eliminate the smell code highlighted by Sonar (too many arguments).
if you consider that clashes a bit with other extensions' implementations, i can put it back.

Reply:
Yes, don't worry about Sonar in this case

Reply:
**EDIT**: I saw that you put a builder. While it is correct, I think you can put a constructor, it is simpler in my opinion for readability, I don't really agree with the Sonar warnings.

Reply:
Done